raiden/raiden_service.py

```diff
@@ -124,6 +129,7 @@ def initiator_init(
         transfer_fee: FeeAmount,
         token_network_identifier: TokenNetworkID,
         target_address: TargetAddress,
+        transfer_secret_hash: SecretHash = None,
```

Comment:
`None` is not a valid value here, also could you please add it just by `transfer_secret`? It's easier to find it that way.

Reply:
done